src/main/resources/css/reporting.css

```diff
@@ -210,6 +210,14 @@ tr:hover {
     color: #eee;
 }
 
+.white-space-normal {
+    white-space: normal;
+}
+
+.white-space-pre-wrap {
+    white-space: pre-wrap;
+}
+
```

Comment:
check if you can reuse some styles from bootstrap library instead of creating new one - if not then add new styles as above